### PR TITLE
Updated from Hybrid Cloud to Universal Print

### PR DESCRIPTION
**Pull request by ChristopherTulip**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/get-ready/printing.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ ms.topic: article
 
 As you get ready to enroll in Microsoft Managed Desktop, you should evaluate your printing requirements and determine the right approach for your environment. You have three options:
  
-- Deploy the Microsoft hybrid cloud print solution to make it easy for Microsoft Managed Desktop devices to discover printers. For more information, see [Deploy Windows Server Hybrid Cloud Print](https://docs.microsoft.com/windows-server/administration/hybrid-cloud-print/hybrid-cloud-print-deploy).
+- Deploy the Microsoft Universal Print solution to make it easy for Microsoft Managed Desktop devices to discover printers. For more information, see [What is Universal Print](https://docs.microsoft.com/en-us/universal-print/fundamentals/universal-print-whatis).
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/printing.md]:
```suggestion
- Deploy the Microsoft Universal Print solution to make it easy for Microsoft Managed Desktop devices to discover printers. For more information, see [What is Universal Print](https://docs.microsoft.com/universal-print/fundamentals/universal-print-whatis).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Comment by yogkumgit** [comment]
@ChristopherTulip Please incorporate the feedback. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks.

**Pull request merged**